File: C2108G1/ADF1/01-10-2022/studentapp/src/main/java/com/aptech/Person.java
package com.aptech;

public class Person {
    private String name;
    private Integer age;
    public Person(){}
    public Person(String name, Integer age) {
        this.name = name;
        this.age = age;
    }

    public void display() {
        System.out.println(this.toString());
    }
    public void input() {

    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getAge() {
        return age;
    }

    public void setAge(Integer age) {
        this.age = age;
    }

    @Override
    public String toString() {
        return "name='" + name +
                ", age=" + age;
    }
}

File: C2009i/AP/27-09-2021/SalaryApp/SalaryApp/Models/Person.cs
using System;
namespace Aptech.Models
{
    public class Person
    {
        public string Name { get; set; }
        public int Age { get; set; }
        public virtual void Display() //virtual = inheritable/can be extended           
        {           
            Console.WriteLine($"name: {Name}, age: {Age}");
        }
        public void Input() //virtual - can be inherit
            
        {
            Console.WriteLine("Enter name: ");
            Name = Console.ReadLine() ?? "";
            Console.WriteLine("Enter age: ");
            Age = Convert.ToInt32(Console.ReadLine());
        }
    }
}

